.github/workflows/build-python.yml

```diff
@@ -93,6 +93,33 @@ jobs:
             langchain_minimal: true
     continue-on-error: false # https://ncorti.com/blog/howto-github-actions-build-matrix
     steps:
+      - name: Check disk space
+        run: |
+          df -h
+      # - name: List installed packages
```

Comment:
Cleanup or we need to keep these steps?

Reply:
@Hartorn I think we can remove the commented out stuff